report/lambda_function.py

```diff
@@ -569,10 +597,19 @@ def handler(event, context) -> None:
     game_df = get_database(conn)
     game_df = format_database_columns(game_df)
 
+    all_emails = get_list_of_emails_from_database(conn)
+
     create_report(game_df, config["DASHBOARD_URL"])
     print("Report created.")
-    send_email(config)
-    print("Email sent.")
+    verification_awaited = []
+    for address in all_emails:
+        try:
+            send_email(config, address)
+            print("Report email sent.")
+        except:
```

Comment:
Maybe be more specific here, is there a specific error we'll get if the user's email isn't verified yet?